pages/BasePage.js

```diff
@@ -0,0 +1,42 @@
+const PAGE_TIMEOUT = 10000
+export default class BasePage {
+
+  /**
+   * Abrir página
+   * @param {String} ruta a la cual acceder
+   */
+  async abrir(ruta) {
+      await browser.url(`${ruta}`);
+  }
+
+  /**
+   * Esperar a que un elemento sea clickeable y hacer click
+   * @param {Object} elemento a clickear
+   */
+  async clickearElemento(elemento) {
+      await elemento.waitForClickable({ timeout: PAGE_TIMEOUT });
+      await elemento.click();
+  }
+
+  /**
+    * Esperar a que un elemento sea clickeable y hacer click
+    * @param {Object} elemento a clickear
+    */
+   async clickearElemento(elemento) {
+    await elemento.waitForClickable({ timeout: PAGE_TIMEOUT });
+    await elemento.click();
+}
+
+  /**
+   * Método para enviar texto a un elemento
+   * @param {Object} elemento que recibirá el texto
+   * @param {String} texto a envíar 
+   */
+  async vaciarCampoYEnviarTexto(elemento, texto){
+      await elemento.waitForClickable({ timeout: PAGE_TIMEOUT });
+      await elemento.clearValue();
+      await elemento.click();
+      await elemento.keys(texto);
```

Comment:
Wdio provides a higher-level api for performing this exact operation: **.setValue()** that will clear current input, click it and send text to it (reducing this from 3 to just 1 line of code). End result simplifies to something like:

```javascript
  async sendText(element, text){
      await element.waitForClickable({ timeout: PAGE_TIMEOUT });
      await element.setValue(text);
```